receiver/mongodbreceiver/metadata.yaml

```diff
@@ -155,6 +163,15 @@ metrics:
       value_type: int
       monotonic: false
     attributes: [database]
+  mongodb.operation.latency.time:
+    description: The latency of operations.
+    unit: "microseconds"
+    enabled: false
+    sum:
+      aggregation: cumulative
+      value_type: int
+      monotonic: false
```

Comment:
I think this should probably be a gauge. We would want it to be a sum only if we expected to add together values in a meaningful way. In this case, let's say we have 100us for reads and 100us for writes, then what does 200us represent?